src/rlx_prv_relup.erl

```diff
@@ -145,7 +155,14 @@ make_upfrom_script(State, Release, UpFrom) ->
                             "relup successfully created!"),
             {ok, State};
         {ok,_, Module,Warnings} ->
-            ?RLX_ERROR({relup_script_generation_warn, Module, Warnings});
+            case WarningsAsErrors of
```

Comment:
ah sorry, i see what you mean, that's a good idea, i've updated the pr accordingly, so now relx deletes the relup file if the `warnings_as_errors' option is set and there are warnings.